File: src/test/java/br/com/blu/bs2/sintetizador/contas/csv/service/utils/RandomicoUtilTest.java
package br.com.blu.bs2.sintetizador.contas.csv.service.utils;

import br.com.blu.bs2.sintetizador.contas.csv.utils.RandomicoUtil;
import org.junit.Before;
import org.junit.Test;

import static org.junit.Assert.assertTrue;

public class RandomicoUtilTest {

    @Before
    public void setUp() {
    }

    @Test
    public void deveGerarValorRandomicoInteiro() {
        System.out.println("#TEST: deveGerarValorRandomicoInteiro: ");

        // -- 01_Cenário && -- 02_Ação
        int valor = RandomicoUtil.gerarValorRandomico();

        // -- 03_Verificacao_Validacao
        assertTrue(valor > 0);
        System.out.println("Valor resultado: ".concat(String.valueOf(valor)));
        System.out.println("-------------------------------------------------------------");
    }

    @Test
    public void deveGerarValorRandomicoLong() {
        System.out.println("#TEST: deveGerarValorRandomicoLong: ");

        // -- 01_Cenário && -- 02_Ação
        Long valor = RandomicoUtil.gerarValorRandomicoLong();

        // -- 03_Verificacao_Validacao
        assertTrue(valor > 0);
        System.out.println("Valor resultado: ".concat(String.valueOf(valor)));
        System.out.println("-------------------------------------------------------------");
    }
}

File: src/test/java/br/com/blu/bs2/sintetizador/contas/csv/service/utils/RandomicoUtilTest.kt
package br.com.blu.bs2.sintetizador.contas.csv.service.utils

import br.com.blu.bs2.sintetizador.contas.csv.utils.RandomicoUtil.gerarValorRandomico
import br.com.blu.bs2.sintetizador.contas.csv.utils.RandomicoUtil.gerarValorRandomicoLong
import org.junit.Assert
import org.junit.Before
import org.junit.Test

class RandomicoUtilTest {
    @Before
    fun setUp() {
    }

    @Test
    fun deveGerarValorRandomicoInteiro() {
        println("#TEST: deveGerarValorRandomicoInteiro: ")

        // -- 01_Cenário && -- 02_Ação
        val valor = gerarValorRandomico()

        // -- 03_Verificacao_Validacao
        Assert.assertTrue(valor > 0)
        println("Valor resultado: $valor")
        println("-------------------------------------------------------------")
    }

    @Test
    fun deveGerarValorRandomicoLong() {
        println("#TEST: deveGerarValorRandomicoLong: ")

        // -- 01_Cenário && -- 02_Ação
        val valor = gerarValorRandomicoLong()

        // -- 03_Verificacao_Validacao
        Assert.assertTrue(valor > 0)
        println("Valor resultado: $valor")
        println("-------------------------------------------------------------")
    }
}
